### DISCUSSION TITLE
How do i submit a guide to be displyed on your website?

### DISCUSSION THREAD

**Discussion by AmIBeingObtuse** [open-webui/open-webui#994] [General]
Hi!

Your website states your asking youtubers to provide guides showing Open WebUI and you may put them on your guide [website](https://docs.openwebui.com/tutorial-deployment/). I have created a guide and wanted to know how I could submit this for consideration. Thanks.

https://www.youtube.com/watch?v=zc3ltJeMNpM

**Comment by tjbck** [comment]
Added for you!

**Reply by AmIBeingObtuse** [reply]
Omg thanks that's fantastic. Really! Thank you very much! 👍😊

Edit: this is an honour. Such a confidence boost. Only started my channel in December.
